tests/assembly/simd-intrinsic-mask-reduce.rs

```diff
@@ -1,6 +1,7 @@
 // verify that simd mask reductions do not introduce additional bit shift operations
 //@ revisions: x86 aarch64
 //@ [x86] compile-flags: --target=x86_64-unknown-linux-gnu -C llvm-args=-x86-asm-syntax=intel
+//@ [x86] compile-flags: -C target-cpu=x86-64
```

Comment:
this is strictly unnecessary, so there is no intuitive reason for us to carry this patch rather than RHEL carrying this patch. which is not a statement that we cannot, but it should not go without comment, I think, as it is not obvious to consult the blame. it would otherwise be "obvious" to remove this.

Reply:
I don't *mind* carrying a patch, and I do for other more hacky things, but we prefer upstream-first where suitable. I'm happy to add a comment to these if that makes it acceptable.